js/main.js

```diff
@@ -12,4 +12,11 @@ const options = {
 };
 
 const game = new Grid(options.gridSize.width, options.gridSize.height, new HTMLRenderer('#snake-game'));
-game.start();
+
+var startButton = document.getElementById('start');
+var resetButton = document.getElementById('reset');
+// game.start();
+
+startButton.addEventListener('click', function () {
```

Comment:
The anonymous function here is a pretty standard pattern but we can do this more directly by passing a reference to the game start function directly:

```js
startButton.addEventListener('click', game.start);
```

That's saying "on a click event, use the game start function" (note the lack of parentheses on `game.start`, indicating that the function is not to be evaluated here but instead will be invoked when the EventListener fires)